src/url/environment.js

```diff
@@ -0,0 +1,3 @@
+const baseUrl = "http://15.206.121.23:7077";
```

Comment:
add v1 also here 
http://15.206.121.23:7077/v1

also folder name should be something like configs instead of url